R/comp_report.R

```diff
@@ -28,8 +28,8 @@
 #' @examples
 #' \dontrun{
 #' #create a PDF report for one sample, as well as exporting all individual plots.
-#' comp_report(this_sample = "HTMCP-01-06-00422-01A-01D",
-#'             out = "reports/",
+#' comp_report(this_sample_id = "HTMCP-01-06-00422-01A-01D",
+#'             out = "./",
```

Comment:
This edit of `out` in the example suggests this can be a recurrent issue. Maybe while on it the `"./"` can become a default value here?